cloudprem/modules/webhooks/charts/connectivity/charts/webhook-service/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: webhook-service
+description: Frontegg Webhook Service Helm chart for Kubernetes
+type: application
```

Comment:
Just to ask, this is basically a copy+paste of the upstream Frontegg Helm chart, including it's deps?

Maybe this gets updated in later commits, but we'll probably want some kind of README to document exactly how to update these charts. That's a notoriously ugly problem, merging charts. Very prone to conflicts. We'll want some breadcrumbs i think to the upsteam source for these charts?

Reply:
Yeah, there's a later commit that does just this: https://github.com/Dozuki/CloudPrem-Infra/pull/3/commits/4d15885adf5d20903770f50203fc55c3653513e4

It might be helpful to have a README on _how_ to update the charts. Even if it's just some links off to where the upstream charts (currently!) live?

Reply:
Sure, good call, we should include the chart update process in the readme. And yea, it's literally just a copy of their charts.